### ISSUE TITLE
Accessibility: Missing autocomplete on form field

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51906]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

This input field collects personal details but does not have a defined autocomplete attribute.  Autocomplete attributes enable browsers to assist people in filling in forms for commonly asked personal information such as name, email address, physical address, phone, credit card details, etc. This is an important accessibility feature that benefits all users.

![Image](https://github.com/user-attachments/assets/58d7262e-ba61-4d8a-9c7d-c08a6262a185)

### Component or Page: 

Page: Product Single (Simple)  

### Name affected element:

Submit review form fields, such as:
1. Name
2. Email

### Relevant code snippets:

```
<input id=""author"" name=""author"" type=""text"" value="""" size=""30"" required="""">

<input id=""email"" name=""email"" type=""email"" value="""" size=""30"" required="""">
```

### Recommended fix:

Add the relevant autocomplete attribute to the fields. For a list of supported autocomplete attributes, see: https://www.w3.org/TR/WCAG21/#input-purposes

**Comment by shameemreza** [comment]
Another similar report: 8844477-zen
